packages/react-vant/src/index-bar/IndexBar.tsx

```diff
@@ -169,7 +169,8 @@ const IndexBar = forwardRef<IndexBarInstance, IndexBarProps>((props, ref) => {
       refs[index].root.current.scrollIntoView();
 
       if (props.sticky && props.stickyOffsetTop) {
-        setRootScrollTop(getRootScrollTop() - props.stickyOffsetTop);
+        setScrollTop(scrollParent, getScrollTop(scrollParent) - props.stickyOffsetTop)
```

Comment:
`setScrollTop` and`setRootScrollTop` is a difference in the definition. needs to be sure that no other problems will arise